scripts/bump-version.sh

```diff
@@ -0,0 +1,9 @@
+VERSION=$1
+
+# Do text substitution in setup.py & README.md
+sed "s/\(version\)=\"[0-9.]*\"\,/\1=\"$VERSION\"\,/" setup.py > tmp
+mv tmp setup.py
+sed "s/\(semgrep\)==[0-9.]*/\1==$VERSION/" setup.py > tmp
+mv tmp setup.py
+sed "s/\(rev\:\)\ 'v[0-9.]*'/\1\ \'v$VERSION\'/" README.md > tmp
+mv tmp README.md
```

Comment:
Why not in-place?

Reply:
https://stackoverflow.com/questions/34533893/sed-command-creating-unwanted-duplicates-of-file-with-e-extension

Reply:
I'm not sure I understand the relevance given this is running on Ubuntu.

Reply:
I think it was just annoying when testing locally + it doesn't really matter that much